WordPress/Classes/ViewRelated/Post/PostCardStatusViewModel.swift

```diff
@@ -216,6 +209,39 @@ class PostCardStatusViewModel: NSObject {
         return [status, sticky].filter { !$0.isEmpty }.joined(separator: separator)
     }
 
+    /// Determine what the failed status message should be and return it.
+    ///
+    /// This is a helper method for `status`.
+    private func generateFailedStatusMessage() -> String {
+        assert(post.remoteStatus == .failed, "This should only be used if the remoteStatus is .failed")
```

Comment:
Somehow this failed for me when offline (there were a post uploading media):

https://cld.wthms.co/PJZ12T

And when debugging:

```
(lldb) po post.remoteStatus == .local
true
```

Reply:
The assertion should have been `post.isFailed`. But I removed it anyway. We'll just rely on the `guard` instead. Done in 77369515f0c31cd21b0455390a24a80b6c6e4e9e.